examples/acquiring_data/fido_metadata_queries.py

```diff
@@ -0,0 +1,48 @@
+"""
+====================================
+Querying Metadata clients using Fido
+====================================
+
+How to perform and inspect metadata only queries
+using Fido from the HEK and HELIO clients.
```

Comment:
This should be expanded, it's quite blunt. Also Fido and the clients should be linked to their classes.

Reply:
Expanded and linked.